enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.tsx

```diff
@@ -143,9 +143,9 @@ const EditSandboxingModal = ({
             <div className="pb2">
               {t`Pick a saved question that returns the custom view of this table that these users should see.`}
             </div>
-            {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-            {/* @ts-ignore */}
+            {/* TODO: remove maxHeight if not needed */}
             <QuestionPicker
+              maxHeight={undefined}
```

Comment:
This error is given incorrectly (due to `@types/prop-types` package) and can be fixed by refactoring `QuestionPicker` to TS.

Reply:
ignored